static/js/controllers/configuration-translation-application.js

```diff
@@ -6,80 +6,96 @@ var _ = require('underscore');
 
   var inboxControllers = angular.module('inboxControllers');
 
-  var findTranslation = function(locale, translation) {
-    var value = _.findWhere(translation.translations, { locale: locale });
-    return value && value.content;
-  };
-
-  var createLanguageModel = function(language, languages) {
-    var rhs = _.find(languages, function(current) {
-      return current.code !== language;
-    });
-    return {
-      lhs: language || 'en',
-      rhs: rhs && rhs.code || 'en'
-    };
-  };
-
-  var createTranslationModels = function(translations, localeModel) {
-    return _.map(translations, function(translation) {
-      return {
-        key: translation.key,
-        lhs: findTranslation(localeModel.lhs, translation),
-        rhs: findTranslation(localeModel.rhs, translation),
-        raw: translation
-      };
-    });
-  };
-
   inboxControllers.controller('ConfigurationTranslationApplicationCtrl',
     function (
       $log,
-      $q,
-      $rootScope,
       $scope,
+      Changes,
+      DB,
       Language,
-      Settings
+      Modal
     ) {
 
       'ngInject';
 
+      var updateLocaleModel = function(language) {
+        var rhs = _.find($scope.translations, function(translation) {
+          return translation.doc.code !== language;
+        });
+        $scope.localeModel = {
```

Comment:
The UI is divided into two columns with one translation on the left and one on the right. This is so someone can translate from the English values to the Spanish values, or from the Spanish values to the French values, or whatever. This UI is the same as before the refactor.